docs/install.md

```diff
@@ -99,27 +99,16 @@ _exact_ same commands presented in the rest of this document.
 We recommend installing with `yarn` or `npm` when:
 
 1. You aren't using a machine with `amd64` or `arm64`.
-2. You're on Linux with `glibc` < v2.17, `glibcxx` < v3.4.18 on `amd64`, `glibc`
+1. You are installing code-server on Windows
+1. You're on Linux with `glibc` < v2.17, `glibcxx` < v3.4.18 on `amd64`, `glibc`
    < v2.23, or `glibcxx` < v3.4.21 on `arm64`.
-3. You're running Alpine Linux or are using a non-glibc libc. See
+1. You're running Alpine Linux or are using a non-glibc libc. See
    [#1430](https://github.com/cdr/code-server/issues/1430#issuecomment-629883198)
    for more information.
 
 Installing code-server with `yarn` or `npm` builds native modules on install.
-This process requires C dependencies; see our guide on [installing these
-dependencies][./npm.md](./npm.md) for more information.
 
-You must have Node.js v12 (or later) installed. See
-[#1633](https://github.com/cdr/code-server/issues/1633).
```

Comment:
the npm.md page recommends node v14 and links to the same issue. Since the VS Code page mentions Node v14, I decided to remove this to be consistent